.github/workflows/phpcs.yml

```diff
@@ -0,0 +1,53 @@
+name: PHPCS
+
+on:
+  pull_request:
```

Comment:
This comes from other Alley projects, and I'm inclined to keep it consistent with those so that it's easier to pick up the Fieldmanager config if you're used to other Alley defaults.